src/Framework/test/SharedFxTests.cs

```diff
@@ -1,29 +1,94 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using Newtonsoft.Json.Linq;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace Microsoft.AspNetCore
 {
     public class SharedFxTests
     {
         private readonly string _expectedTfm;
+        private readonly string _expectedRid;
+        private readonly ITestOutputHelper _output;
 
-        public SharedFxTests()
+        public SharedFxTests(ITestOutputHelper output)
         {
+            _output = output;
             _expectedTfm = "netcoreapp" + TestData.GetPackageVersion().Substring(0, 3);
+            _expectedRid = TestData.GetSharedFxRuntimeIdentifier();
+        }
+
+        [Fact]
+        public void PlatformManifestListsAllFiles()
+        {
+            var platformManifestPath = Path.Combine(TestData.GetManifestOutputDir(), "Microsoft.AspNetCore.App.PlatformManifest.txt");
+            var expectedAssemblies = TestData.GetSharedFxDependencies()
+                .Split(';', StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Is `RemoveEmptyEntries` really needed? Or would it just mask potential bugs where we put blank entries in the metadata?

Reply:
Not in this case. I'm only using remove empty entries on TestData.GetSharedFxDependencies(), which is test data generated in the unit test csproj file. The data isn't used in the product.